src/osal/printf.c

```diff
@@ -33,7 +33,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#include <printf.h>
+#include <osal/printf.h>
```

Comment:
It applies to all boards. We can talk about this more but the point of the HAL / OSAL is to hide away the hardware details. If something is defined at this layers (or above) then it's "supported" by all boards. But supported just means that it compiles and you can call it. It doesn't guarantee it actually prints. For example, printf does nothing for wavious-mcu, but it will still compile. This means that the same code can be written for different boards and those that actually print (via serial, uart, etc.) will produce output.